LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1782,6 +1782,18 @@ webkit.org/b/261356 [ Debug ] fast/mediastream/device-change-event-2.html [ Pass
 webkit.org/b/263396 css3/color/text.html [ Skip ]
 
 # WebGPU
+
+[ Release ] http/tests/webgpu/webgpu/api/operation/buffers/map.html [ Pass ]
```

Comment:
These really don't seem correct?
Tests should pass in Debug and Release by default.

Reply:
A lot of the CTS tests timeout in Debug as they take longer than the slow timeout, maybe not this one, I will double check